pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -44,24 +44,64 @@ Quick link:
 
 ## Introduction {#intro}
 
-<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is a W3C Group Note that provides informative guidance (not normative requirements).
+<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is the acronym used for the W3C Guidance on Applying WCAG 2 to Non-web Information and Communication Technology Group Note.
+
+The WCAG2ICT Group Note provides:
+* [Informative](https://www.w3.org/TR/WCAG22/#dfn-informative) **guidance** that is not [normative](https://www.w3.org/TR/WCAG22/#dfn-normative) and that does not set requirements.
+* **Comments on applicability** of the Web Content Accessibility Guidelines (WCAG) when applied to non-web information and communications technologies (ICT) - this includes  non-web documents and software, including mobile apps, native applications, platform software, and [software on ICT with closed functionality[(https://www.w3.org/TR/wcag2ict/#comments-on-closed-functionality).
+* **Technology-specific explanations or difficulties** for some success criteria. This is especially the case where WCAG makes assumptions on the presence of a browser that would not be true for non-web documents and software. Not all ICT have these concepts.
+
+The WCAG2ICT Group Note is intended to help clarify how to use WCAG 2 to make non-web documents and non-web software more accessible to people with disabilities.  Addressing accessibility involves addressing the needs of people with auditory, cognitive, neurological, physical, speech, and visual disabilities, and the needs of people with accessibility requirements due to the effects of aging.  Although this document covers a wide range of issues, it is not able to address all the needs of all people with disabilities.
```

Comment:
```suggestion
The WCAG2ICT Group Note is intended to help clarify how to use WCAG 2 to make non-Web documents and software more accessible to people with disabilities. Addressing accessibility involves addressing the needs of people with auditory, cognitive, neurological, physical, speech, and visual disabilities, and the needs of people with accessibility requirements due to the effects of aging. Although this document covers a wide range of issues, it is not able to address all the needs of all people with disabilities.
```

Reply:
Same comment regarding "non-Web" vs. "non-web". It's confusing on how to know which form should be used.